src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Clearbooks\Labs\Toggle\Entity;
 
-interface ActivatableToggle
+interface ActivatableToggle extends Toggle
```

Comment:
Another approach is to have two interfaces Activatable and Toggle

then your concrete Toggle entity object can just implement both of the interfaces. Which would be a neater solution as @viktorcb suggested.